### ISSUE TITLE
Add CI integration

### ISSUE THREAD

**Issue by michaelbukachi** [thealmikey/jiplace-android#1]
Hey. Now that you've made this project public you should setup CI and a status badge. People like seeing green:joy:. It motivates people to take a look into the project.

**Comment by normaneshuchi** [comment]
That's super! I'll pull the largest changes and build and try it out....
Otherwise, good work.

**Comment by thealmikey** [comment]
Definitely, i'll set up some Travis, feed the project some greens😂 make it healthy😂😂
